### PR TITLE
feat: add sveltekit ts project example

### PR DESCRIPTION
**Pull request by indaco**
This PR adds a sample project to showcase how to use maxgraph with SvelteKit

### CODE REVIEW

**Review by tbouffard — COMMENTED** [review]
Hi @indaco thanks for providing this example
I have limited time so I only did a quick review. This looks very promising 💯 

I will test it in the few days.
In the meantime, can you check the first comments I already wrote?

File: projects/sveltekit-ts/src/routes/+layout.svelte
[the remark this answers is not in the document]
Comment (indaco) [reply projects/sveltekit-ts/src/routes/+layout.svelte:1]: Yes, it is

File: README.md [hunk 1]
```diff
@@ -2,41 +2,43 @@
 
 Demonstrate how to integrate [maxGraph](https://github.com/maxGraph/maxGraph/) in projects.
 
-⏩ Find the live demo at https://maxgraph.github.io/maxgraph-integration-examples/
+⏩ Find the live demo at <https://maxgraph.github.io/maxgraph-integration-examples/>
```
[the remark this answers is not in the document]
Comment (indaco) [reply README.md:5]: Will do

File: projects/sveltekit-ts/src/app.d.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+// See https://kit.svelte.dev/docs/types#app
+// for information about these interfaces
+declare global {
+	namespace App {
+		// interface Error {}
+		// interface Locals {}
+		// interface PageData {}
+		// interface Platform {}
```
[the remark this answers is not in the document]
Comment (indaco) [reply projects/sveltekit-ts/src/app.d.ts:8]: That file is autogenerated by the _create-svelte_ project template and it is standard in sveltekit project.

[the remark this answers is not in the document]
Comment (tbouffard) [reply projects/sveltekit-ts/src/app.d.ts:8]: Fair enough, so forget about this comment

**Review by tbouffard — APPROVED** [review]
Thanks for this contribution.
I made some minors updates for consistency with other projects.

This PR already covers #41.

**Comment by indaco** [comment]
@tbouffard thanks for merging 

I cannot see the live demo page including it

Is there anything you need for that?

**Pull request merged**